src_python/habitat_sim/robots/mobile_manipulator.py

```diff
@@ -274,15 +274,17 @@ def reset(self) -> None:
     @property
     def arm_joint_limits(self) -> Tuple[np.ndarray, np.ndarray]:
         """Get the arm joint limits in radians"""
-        arm_pos_indices = list(
-            map(lambda x: self.joint_pos_indices[x], self.params.arm_joints)
-        )
 
+        # deref self vars to cut access in half
+        joint_pos_indices = self.joint_pos_indices
+        joint_limits = self.joint_limits
+        arm_joints = self.params.arm_joints
+        arm_pos_indices = [joint_pos_indices[x] for x in arm_joints]
         lower_lims = np.array(
-            [self.joint_limits[0][i] for i in arm_pos_indices], dtype=np.float32
+            [joint_limits[0][i] for i in arm_pos_indices], dtype=np.float32
```

Comment:
@jturner65 You may want to split joint_limits[0] into two cached local variables where you are at it here.